[#997] README / Home Page: Describe RepoSense features from users' perspective

anubh-v opened the pull request:
Fixes #997 

```
README: Describe RepoSense features from users' perspective

While the README / Home Page have a list of RepoSense's capabilities,
this might not sufficiently convey the value of using RepoSense.
Let's frame RepoSense's features from the
users' perspective. This will help readers appreciate the product.

Let's also make some minor grammar fixes.
```

Tejas2805 commented on README.md:
```diff
@@ -5,8 +5,10 @@
-The features of the report includes:
+RepoSense is a contribution analysis tool for Git repositories. It is particularly useful for those who want a chronological visualisation of the code contributions and commits made to a group of repositories. Specifically, educators can use RepoSense to track the work done in student projects, while team leads may use it to oversee their project teams.
+
+RepoSense generates a static HTML report that includes contribution information for each author in each target repository.
```
Will **each target user** be more appropriate?

anubh-v replied:
I think it is a bit too verbose to say `each target user in each target repository`.

Tejas2805 commented:
@anubh-v Is this PR ready for review?

anubh-v commented:
@Tejas2805 thanks for looking through this PR. I'll update it shortly.

anubh-v commented:
Ready for reviews

jamessspanggg commented on README.md:
```diff
@@ -5,8 +5,10 @@
 
-RepoSense is a contribution analysis tool for Git repositories. It generates a static HTML report including contribution information for each author in the repository.
-The features of the report includes:
+RepoSense is a contribution analysis tool for Git repositories. It is particularly useful for those who want a chronological visualisation of the code contributions and commits made to a group of repositories. Specifically, educators can use RepoSense to track the work done in student projects, while team leads may use it to oversee their project teams.
```
code contribution instead of code contributions?

anubh-v replied:
I think it should be "contributions" since it's plural, right?

jamessspanggg replied:
haha yep you're right

eugenepeh replied:
i think we can include more group of users e.g. project managers.

anubh-v commented:
@fzdy1914 may I get your review on this? Thanks for your time

fzdy1914 commented:
Better ask a review from @eugenepeh  for document work.

damithc commented:
> do seek @damithc 's opinion too

No objections to the current version.

anubh-v commented:
> should we also include a gif to show case some of the value add functionalities?

Sure, I think this is a good idea.
We could use the dashboard from one of the modules to illustrate some features.
I'll add it in within this PR?

eugenepeh commented:
>Sure, I think this is a good idea.
We could use the dashboard from one of the modules to illustrate some features.
I'll add it in within this PR?

Yes please, can reuse/redo one of the gif from user guide for this.

anubh-v commented:
@eugenepeh, I have added a GIF showing some useful features. May I get your review?

Here is the GIF that I added:
![overview](https://user-images.githubusercontent.com/35621759/81132361-beecd780-8f80-11ea-93dc-984bfd82936e.gif)

damithc commented:
> @eugenepeh, I have added a GIF showing some useful features. May I get your review?

Let's minimize the use of GIFs as they are hard to update incrementally when the UI changes.

anubh-v commented:
@damithc, thanks for the response.

> Let's minimize the use of GIFs as they are hard to update incrementally when the UI changes.

Ok. I have removed the new GIF. Instead, I reused the images and the single GIF that are already present in the User Guide.

I have used the Tabs feature in Markbind to showcase the images / GIF.

anubh-v commented:
@fzdy1914 could this PR be merged?
I've addressed @eugenepeh 's comment by including some images / GIF in the Home Page.
